.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,3 +1,12 @@
+---
+name: Bug report
+about: Create a report to help us improve.
+title: ''
+labels: kind/bug
+assignees: ''
+
```

Comment:
Is there a reason for this extra empty line here and in the other file?

Reply:
I generated the files with the Github Settings helper.

Reply:
Ah ok, maybe it adds an extra empty line, oh well :man_shrugging:

Reply:
It works if I remove the line.